internal/web/templates/pages/packages.html

```diff
@@ -5,30 +5,32 @@
       <div class="col">
         <div class="card h-100 border-primary border-1">
           <div class="card-body p-0 d-flex flex-column">
-            <div class="d-flex align-items-center">
+            <div hx-boost="true" class="d-flex align-items-center">
               <div class="flex-shrink-0 px-1 py-1 align-self-center">
-                {{if eq .IconUrl ""}}
-                <!-- TODO the glasskube logo as fallback is probably not the best idea? -->
-                <img src="static/assets/glasskube-logo.svg" alt="{{ .PackageName}}"  style="max-width:52px; max-height: 52px; height:auto">
-                {{else}}
-                <svg width="52" height="52" style="max-width: 52px; max-height: 52px; height: auto;">
-                  <image xlink:href="{{ .IconUrl }}" width="52" height="52" alt="{{ .PackageName}}" />
-                </svg>
-                {{end}}
+                <a class="text-decoration-none" href="/packages/{{ .PackageName }}">
+                  {{if eq .IconUrl ""}}
+                  <!-- TODO the glasskube logo as fallback is probably not the best idea? -->
+                  <img src="/static/assets/glasskube-logo.svg" alt="{{ .PackageName}}"  style="max-width:52px; max-height: 52px; height:auto">
+                  {{else}}
+                  <img src="{{ .IconUrl }}" alt="{{ .PackageName}}"  style="max-width:52px; max-height: 52px; height:auto">
```

Comment:
I want to have a look at this svg thing again, because it seems to me that it changed the design on the package overview page a bit (in the case of cyclops with the short short description, the logo + text are more horizontally centered instead of at the top). I also have to fix it on the package detail page

Reply:
should be fixed now